app/models/profile_field.rb

```diff
@@ -20,7 +23,7 @@ class ProfileField < ApplicationRecord
   validates :display_area, presence: true
   validates :input_type, presence: true
   validates :label, presence: true, uniqueness: { case_sensitive: false }
-  validates :show_in_onboarding, inclusion: { in: [true, false] }
```

Comment:
This column is non-nullable so the boolean validation can go.

Reply:
Lately, we've been leaning towards not adding these or even removing them. Right now this value only gets populated via a checkbox in a form in the admin section, which we fully control. I'm not sure how that would return a value other than true or false. 

I have no strong feelings about this, I can add it back in given that it isn't directly related to this PR.

Reply:
@citizen428 Yeah, it's hard to know which to do, tbh. I could argue that we don't know what all will provide this value in the future (for example, we could add an API endpoint for it) and you could argue that I'm overthinking it/YAGNI and we'd both be right. :sob:

Reply:
I added it back in for now since it isn't related to this PR anyway. We can maybe make a more comprehensive call in the future and then employ the same strategy across all models with boolean columns.